### DISCUSSION TITLE
Feature request: take over Ctrl+S event

### DISCUSSION THREAD

**Discussion by wohenbushuang** [outline/outline#3687] [Ideas]
It's a conventional habit to press Ctrl+S during writing, but everytime browser will pop up the save webpage dialog.
What about let outline take over the Ctrl+S event, and trigger a synchronization with the host?

**Comment by tommoor** [comment]
Hmm weird, we should already be capturing this while you have the editor focused – on macOS at least this works as expected

**Reply by wohenbushuang** [reply]
Oh thx, now I know why. It indeed also works on windows+chrome.
The reason the save webpage dialog will pop up is that the capture only effects when the cursor is inside the editor.
But sometimes, I press Ctrl+S after clicking the table of contents...
